### ISSUE TITLE
Subdirectory names in paths of files are not created

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_bagger#31]

**Comment by mjordan** [comment]
Now if you use a directory in a filename specified in your config file, like this:

`templated_output_filename: 'metadata/MODS.xml'`

The directories in the path will be created. More work to do.

**Comment by mjordan** [comment]
In AddMedia plugin, this works `$bag->addFile($temp_file_path, 'foo/' . basename($temp_file_path));
`.

**Comment by mjordan** [comment]
Now you can use this in your config file:

```
# Used by the 'AddMedia' plugin. Include the trailing /.
media_file_directories: 'foo/bar/baz/'
```
An the media files will be placed in `data/foo/bar/baz/media.png` in the Bag.

**Comment by mjordan** [comment]
Resolved with 1be9fa3adbc7352be1769f1f08b3a93b83ab24b9.
